aten/src/ATen/core/dispatch/DispatchTable.h

```diff
@@ -68,6 +68,7 @@ class KernelTable_ final {
 
   void removeIfExists(TensorTypeId key, const std::string& operator_name) {
     auto num_removed = map_.erase(key);
+    (void)num_removed;
     assert(num_removed <= 1); // This is not a multi-map
```

Comment:
Easier fix is to stop using `assert` here; we should use `TORCH_INTERNAL_ASSERT`

Reply:
Done.